js/jsmodules.go

```diff
@@ -46,13 +46,18 @@ func getInternalJSModules() map[string]interface{} {
 			"k6/experimental/timers is now part of the k6 core, please change your imports to use k6/timers instead."+
 				" The k6/experimental/timers will be removed in k6 v0.52.0"),
 		"k6/experimental/tracing": tracing.New(),
-		"k6/experimental/browser": browser.NewSync(),
-		"k6/experimental/fs":      fs.New(),
-		"k6/net/grpc":             grpc.New(),
-		"k6/html":                 html.New(),
-		"k6/http":                 http.New(),
-		"k6/metrics":              metrics.New(),
-		"k6/ws":                   ws.New(),
+		"k6/experimental/browser": newWarnExperimentalModule(browser.NewSync(),
+			"Please update your imports to use k6/browser instead of k6/experimental/browser,"+
+				" which will be removed after September 23rd. Ensure your scripts are migrated by then."+
```

Comment:
I'd suggest to be explicit about that date

```suggestion
				" which will be removed after September 23rd, 2024. Ensure your scripts are migrated by then."+
```